drop-in/src/main/java/com/adyen/checkout/dropin/ui/action/ActionComponentDialogFragment.kt

```diff
@@ -151,11 +161,16 @@ internal class ActionComponentDialogFragment : DropInBottomSheetDialogFragment()
     }
 
     private fun handleError(componentError: ComponentError) {
-        when (componentError.exception) {
+        when (val exception = componentError.exception) {
             is CancellationException -> {
                 Logger.d(TAG, "Flow was cancelled by user")
                 onBackPressed()
             }
+            is PermissionException -> {
+                val requiredPermission = exception.requiredPermission
+                Logger.e(TAG, "$requiredPermission permission is not granted", exception)
```

Comment:
Then here you can simply log the message from the exception itself since it should contain the formatted permission name